### PR TITLE
fixed module not found error in Dockerfile + Misc updates

### PR DESCRIPTION
**Pull request by codecakes**
Fixes issue with running docker on this project. `docker compose up --build` works now.
- updated Readme 
- removed unnecessary code in init. 
- Upgraded pylint version. 
- generated new pylintrc because of this [PR](https://github.com/PyCQA/pylint/pull/4942)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
+RUN apt-get update -y && apt install -y libsqlite3-dev
 RUN python3 -m pip install -U pip && python3 -m pip install --no-cache-dir poetry
 
-RUN rm -rf *
+WORKDIR /app
 
-#WORKDIR /app
-
-COPY registrations /app
+ADD registrations /app/registrations
```
[the remark this answers is not in the document]
Comment (Matt-Murungi) [reply Dockerfile]: From my understanding, the issue was that the "registration" module was not being found everytime one would run docker build. 

I tried it on my local and it worked. So LGTM.

**Review by Matt-Murungi — COMMENTED** [review]
From my understanding, the issue was that the "registration" module was not being found every time one would try to build the docker image.

I tried it on my local and it worked. So LGTM @codecakes

**Comment by codecakes** [comment]
Thanks @Matt-Murungi could you approve?
> 
> 
> From my understanding, the issue was that the "registration" module was not being found every time one would try to build the docker image.
> 
> 
> 
> I tried it on my local and it worked. So LGTM @codecakes

**Review by mmurungi — APPROVED** [review]
LGTM!

**Pull request merged**